### ISSUE TITLE
Corber is swallowing all cordova errors

### ISSUE THREAD

**Issue by kbrownlees** [isleofcode/corber#617]
Not sure if this is my setup or just how it works, but quite a few times the underlying cordova calls have been failing but corber is swallowing them so there is no indication that anything is wrong.

eg:

```
$> corber build --verbose --platform ios

... snip ...

[!] The Podfile does not contain any dependencies.

Linting corber/cordova/www/index.html...

0 problems
corber project built
```

But the actual build is failing:

```
$> cordova build --debug ios

... snip ...

[!] The Podfile does not contain any dependencies.
Reading build config file:
CordovaError: Promise rejected with non-error: 'ios-deploy was not found. Please download, build and install version 1.9.2 or greater from https://github.com/ios-control/ios-deploy into your path, or do \'npm install -g ios-deploy\''
    at cli.catch.err (/usr/local/lib/node_modules/cordova/bin/cordova:29:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

I do have a custom framework, but it was mainly copied from the react framework.

**Comment by c-emil** [comment]
I think there should be a flag available, in order to tell corber to display all cordova output. I had similar issues. I was installing a platform and corber said nothing about failure, whereas when I tried `cordova platform add ios`, cordova warned me about various plugin failures. For example, cordova-plugin-intercom didn't install because of missing Cocoapods or another plugin didn't install because a certain version that I specified wasn't found on server. 

If not all errors are shown by default, cober should at least warn user that there was a failure while adding a platform and that the user can call the command once again with this flag that will allow the user to show the errors.
